testplan/runners/pools/base.py

```diff
@@ -330,10 +331,9 @@ def handle_request(self, request):
         sender_index = request.sender_metadata['index']
         worker = self._workers[sender_index]
         if not worker.active:
-            self.logger.critical(
-                'Ignoring message {} - {} from inactive worker {}'.format(
+            self.logger.warning(
```

Comment:
To be honest I'm not sure this should be a warning - I think it is quite expected when processing a message that the worker may have died. Let's leave as a warning for now but if it gets too spammy we should reduce to a debug log.